speech.d.ts

```diff
@@ -57,26 +122,34 @@ declare module 'lmnt-node' {
     finish(): void;
 
     /**
-     * Returns an async iterator that yields synthesized speech audio data
-     *     as a 96kbps mono MP3 stream with a sampling rate of 24kHz.
+     * Returns an async iterator that yields an object containing synthesized speech 
+     * audio data as a 96kbps mono MP3 stream with a sampling rate of 24kHz.
      */
-    [Symbol.asyncIterator](): AsyncIterableIterator<AudioData>;
+    [Symbol.asyncIterator](): AsyncIterableIterator<Object>;
   }
 
   type Voice = {
     id: string;
     name: string;
-    gender: string;
-    imageUrl: string;
+    owner: string;
     state: string;
-    tags: string[];
+    type: string;
```

Comment:
We could probably specify this (and gender below) more explicitly like you did for 'owner' at line 52 above in this PR to say it's one of either "instant" or "professional". The same is however true for 'state' above... and doing so would lose future flexibility. So I think don't make the change, but I wanted to mention it while reading.

Reply:
I think I will do for type, discussion to be had for gender. It's kind of a useless tag at the moment but I imagine we could filter by gender in the future. discussion to be had around what to enum as specifically to be sensitive. although male, female, nonbinary, other feels pretty exhaustive to me